nemo/core/optim/lr_scheduler.py

```diff
@@ -270,6 +277,7 @@ def __init__(
         self.decay_steps = max_steps - (self.constant_steps + self.warmup_steps)
 
         self.min_lr = min_lr
+        self.first_step = True
```

Comment:
can you explain what this flag does?

Reply:
@yaoyu-33 it's needed to reset self.max_steps and self.decay_steps when reset_lr_steps is True.

Reply:
we only need to reset them once at the beggining of the training